pages/dkp/kommander/2.2/cli/dkp/completion/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: layout.pug
+navigationTitle:  dkp completion
+title:  dkp completion
+menuWeight: 10
+excerpt: Generate the autocompletion script for the specified shell
+notes: Automatically generated, DO NOT EDIT
+enterprise: false
+beta: false
+---
+<!-- vale off -->
+<!-- markdownlint-disable -->
+
+## dkp completion
+
+Generate the autocompletion script for the specified shell
```

Comment:
I know these are automated, but Is it possible to list the targets of this command (i.e. zsh, fish, etc) here?  I see in other commands we have 'one of [xxxx]',    I notice also that there are help links for all the possibilities, so it seems odd that they are not listed here.

Reply:
This is part of the CLI library we use. Might be possible to override though (in the `dkp-cli` repo, will show up here as an automated PR later).